src/dbnode/persist/fs/retriever_test.go

```diff
@@ -279,7 +346,13 @@ func testBlockRetrieverHighConcurrentSeeks(t *testing.T, shouldCacheShardIndices
 
 					require.NoError(t, err)
 					compare.Head = shardData[r.shard][r.id][xtime.ToUnixNano(r.blockStart)]
-					assert.True(t, seg.Equal(&compare))
+					require.True(
+						t,
+						seg.Equal(&compare),
+						"data mismatch for series %s, returned data: %v",
```

Comment:
Looks like you have only two printf wildcards but three vars that follow?

I find to avoid this possibility it's actually easier to just wrap this up in it's own `fmt.Sprintf(...)` to `require` or `assert`.